src/BuffLog/BuffLog.php

```diff
@@ -86,6 +92,18 @@ public static function __callStatic($methodName, $args)
         }
     }
 
+    private static function checkDebug()
+    {
+        if (is_array($_GET) && isset($_GET[self::$debugKey])) {
```

Comment:
should we take into account only `GET` requests or  also `POST` ones?

Reply:
I think POST too - particularly for API requests (eg creating an update) it would be useful to get the trace in those situations too as well as the 'read' requests.